Klipper_Files/ercf_software.cfg

```diff
@@ -815,9 +815,7 @@ gcode:
     {% if printer["gcode_macro _ERCF_PAUSE"].is_paused|int == 0 %}
         {% set selector_homed = 0 %}
         {% if printer["gcode_macro _ERCF_VAR"].sensorless_selector|int == 1 %}
-            {% if printer.query_endstops.last_query["manual_stepper gear_stepper"] == 1 %}
```

Comment:
I found this line to be a bit strange, since we use a fake endstop here previously the only effect is to always pass the check. Since the sensorless endstop isn't persistently active once triggered, we cannot check the selector endstop, I think the only option is to assume it homed in all cases.